zanshincli/main.py

```diff
@@ -1041,6 +1040,31 @@ def scan_target_groups_create_by_compartments(
     client = Client(profile=global_options['profile'])
     dump_json(client.create_scan_target_by_compartments(organization_id, scan_target_group_id,name, ocid))
 
+@scan_target_group_app.command(name='update')
+def scan_target_groups_update(
+        organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+        scan_target_group_id: UUID = typer.Argument(..., help="UUID of the scan target group"),
+        name: str = typer.Argument(..., help="new name of the scan target group")
+):
+    """
+    Updates a scan target group.
+    """
+    client = Client(profile=global_options['profile'])
+    dump_json(client.update_scan_target_group(organization_id, scan_target_group_id,name))
+
+@scan_target_group_app.command(name='create')
+def scan_target_groups_create(
+        organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+        kind: ScanTargetKind = typer.Argument(..., help="kind of the scan target group. Should be 'ORACLE'" ),
+        name: str = typer.Argument(..., help="name of the scan target group")
+):
+    """
+    Create a scan target group of the organization.
```

Comment:
Creates

(Need to be consistent with others. Either all in third person or all not)